src/delay/readme.md

```diff
@@ -52,7 +52,7 @@ It is useful when you know that calculations requires more time if you have more
 ### Formulae
 
 ```ts
-target = delay({ source, timeout: Function, target });
+event = delay({ source, timeout: Function, target });
```

Comment:
```suggestion
target = delay({ source, timeout: Function, target });
```

Here we need the same name as `target` in arguments, to show it is the same reference.